e2e/test/scenarios/dashboard/dashboard-back-navigation.cy.spec.js

```diff
@@ -104,8 +106,8 @@ describe("scenarios > dashboard > dashboard back navigation", () => {
     { tags: "@slow" },
     () => {
       const cardTitle = "Sales per state";
-      cy.visit(`/auto/dashboard/table/${ORDERS_ID}`);
-      cy.wait("@dataset");
+      cy.visit(`/auto/dashboard/table/${ORDERS_ID}?show=${MAX_CARDS}`);
```

Comment:
I can't get this `show` query param to do anything 🤔

Reply:
Did some digging into the API vs. code code. Core uses `show` to limit cards, but the API doesn't actually pass it through, so this isn't a good fix ATM. WDYT of upping the timeouts on those two calls? That _seems_ to be the issue.

Reply:
Pushed a change and now it should work.

Reply:
Upping the timeouts is probably a fine band-aid to reduce the flakes but I've seen some of these API requests hit a minute or more, so it doesn't really address the issue.

Reply:
I suspect the timing is just because of the amount of time required to hit every endpoint for every dashcard. I think only showing a few cards should fix this. From a product perspective, perhaps someone only wants to see the top 2-3 cards.